tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -339,7 +340,7 @@ def _wait_for_toggle_ready(self, by, locator):
         Wait(self.marionette).until(expected.element_enabled(checkbox))
 
     def return_to_prev_menu(self, parent_view, exit_view):
-        GaiaHeader(self.marionette, self._header_locator).go_back()
+        GaiaHeader(self.marionette, exit_view.find_element(*self._header_locator)).go_back()
```

Comment:
What is the difference between the old and the new line? If I understand the `GaiaHeader` constructor correctly, we're passing the element instead of the locator, right?